kevm-pyk/src/tests/unit/test_unit.py

```diff
@@ -230,3 +231,20 @@ def test_contract_creation() -> None:
     assert method.arg_types == ('tuple[]',)
     assert method.contract_name == 'TestContract'
     assert method.payable
+
+
+def test_int_to_hex() -> None:
```

Comment:
Please refactor this to a [parametric test](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions).

Test data can be a list of pairs `(input, expected)` so that you can assert equality with the actual value.

Reply:
Also, please add tests for composite `KAst` objects.

Reply:
Done!